lib/ssf/client.rb

```diff
@@ -27,69 +27,25 @@ def send_to_socket(message)
       @socket.send(message, 0)
     end
 
-    def new_root_span(operation: '', tags: {})
+    def start_span(operation: '', tags: {}, trace_id: nil, parent_id: nil)
```

Comment:
We'd probably want to name that some derivative of `extract`, on second thought, since that's more analogous to the opentracing stuff.